### DISCUSSION TITLE
Jamulus  / Jack / Voicemeeter - get Jamulus output to PC

### DISCUSSION THREAD

**Discussion by leoinlios** [jamulussoftware/jamulus#2108] [Technical Support & Help]
Hi, I am stuck guys. Help would be much appreciated. 
Setup: Win10 PC - TD-27 drum module <USB> PC + Mic <USB> PC. 
I have done this so far using Jack, Voicemeeter, Jamulus, as per https://jamulus.io/kb/2020/11/24/Multiple-Audio-Interfaces.html: 
https://user-images.githubusercontent.com/4964741/140050954-b69ec233-adad-4987-b11b-bad79057e3a0.png
This allows me to send my PC mic to Jamulus and my drum module to Jamulus and I get great latency. 
Now I need to be able to **send Jamulus out to the PC headphones** so that I can here the server sound. 

Why not send the output to the drum module? Because I want to follow the Golden rule and the drum module cannot stop direct monitoring, so headphones need to be in the PC. 

I know Voicemeeter without JACK can do what I describe above but latency is terrible. 
I know I can also use an external interface, but I purchased the Presonus Studio 24c and the sound I hear from the server, compared to how my TD-27 sounds directly - is terrible too. 
If I can somehow **route back Jamulus into Voicemeeter**, I would be sorted, but Voicemeeter does not see Jamulus as an Input...

Thanks for your help.

**Comment by jujudusud** [comment]
Do you have the possibility to use GNU/Linux instead of Windows ?
Jamulus OS is a ready to use distribution :
https://sourceforge.net/p/jamulus-os/wiki/Home/

**Reply by leoinlios** [reply]
Feels a bit extreme to have to change OS when I am half way there...is there no solution to this in Windows?

**Reply by rdica** [reply]
> Do you have the possibility to use GNU/Linux instead off Windows ? Jamulus OS is a ready to use distribution : https://sourceforge.net/p/jamulus-os/wiki/Home/

Is this even still in development? The first step is to download what is now a non-existent file. Doesn't bode well for being maintained.

**Reply by pljones** [reply]
It's also a bit dodgy putting a copy of the UbuntuStudio ISO image in their own repo, rather than linking to the UbuntuStudio repo.

**Reply by jujudusud** [reply]
> Feels a bit extreme to have to change OS when I am half way there...is there no solution to this in Windows?

Yes, a bit extreme... There are solutions in Windows 10, but not for with the latency you want to achieve.

You will be able to use an ASIO audio interface (Your Presonus) where you will connect, in audio not usb, your drum and your microphone and also your monitors or your headphones. I think it is the only low latency way to do that.

**Reply by leoinlios** [reply]
Using an interface with analogue input from drums and analogue input from mic is the basic form, this thread is about: **send Jamulus out to the PC headphones**, precisely so that I don't have to use such external interface.

**Reply by jujudusud** [reply]
Understood.
You have to use only one audio interface for all the inputs and outputs to achieve a low latency.
If the Pc headphone is connected to the internal audio interface, then you have to connect all the inputs to this interface too.

**Reply by leoinlios** [reply]
ok thanks

**Comment by pcar75** [comment]
Bonjour à tous, 

Sadly Jamulus OS sources were removed , see [https://sourceforge.net/p/jamulus-os/discussion/v2/thread/844388daa3/](url) . Most of the documentation still remains though.

I use it booting from a USB key , so as to shave from 4 to 10 ms of overall delay (better than my regular W10 boot).  From the included software of that 're-distribution', I've easily added the included VLC player as a (PulseAudio) link with the jack connection kit (QjackCtl), so I can play some less known songs to fellow musicians, while I play guitar over it.

I use the Patchbay or (mostly) the Graph function to connect/disconnect sources and destinations. I have not tried connecting multiple Jamulus outputs since I run my sound through a small mixer, but believe it's feasible.
Ex.: 
![JamulusOS_QjackCtl currated Graph with VLCPlayer](https://user-images.githubusercontent.com/64435162/140182020-859435a1-711f-40a2-865d-60d05a025d7c.png)

**Reply by leoinlios** [reply]
Hi - not sure what you are adding, this thread is about connecting Jamulus so that **its server sound comes out in the PC**, not in an interface. Is it possible?

**Reply by pcar75** [reply]
An answer to a suggestion @rdica  made above, with some comments as to how Jack Audio Connection Kit (Linux, but also Windows) can be used ; pretty versatile, I think.
